ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/home/viewmodel/WalletViewModel.kt

```diff
@@ -267,7 +330,9 @@ private fun Synchronizer.toWalletSnapshot() =
             orchardBalance ?: WalletBalance(Zatoshi(0), Zatoshi(0)),
             saplingBalance ?: WalletBalance(Zatoshi(0), Zatoshi(0)),
             transparentBalance ?: WalletBalance(Zatoshi(0), Zatoshi(0)),
-            pendingCount
+            pendingCount,
+            flows[6] as Int,
```

Comment:
Here is where we'd probably do the conversion to PercentDecimal that I mentioned earlier.

Eventually, I'd like to move that to the SDK as part of the type safe APIs.

Reply:
I filed https://github.com/zcash/zcash-android-wallet-sdk/issues/590 in the SDK.